cmd/traffic_manager/traffic_manager.cc

```diff
@@ -318,7 +318,7 @@ init_dirs()
 static void
 chdir_root()
 {
-  const char *prefix = Layout::get()->prefix;
+  const char *prefix = Layout::get()->prefix.c_str();
```

Comment:
Isn't there a risk of the temporary from `Layout::get()` going out of scope before `prefix` is used?